Graphs/DisjointSetUnion.java

```diff
@@ -0,0 +1,60 @@
+import java.io.*;
+import java.util.*;
+class DisjointSet {
```

Comment:
add brief about DSU and wiki link under comments here.

Reply:
added